crates/web-plugins/did-endpoint/src/plugin.rs

```diff
@@ -36,6 +35,18 @@ fn get_env() -> Result<DidEndpointEnv, PluginError> {
         server_public_domain,
     })
 }
+pub fn get_master_key<'a>() -> Result<String, PluginError> {
+    let master_key = std::env::var("MASTER_KEY").expect("Secrets Mastet_KEY env variable required");
```

Comment:
Bubble up the error instead of panicking. We're doing that for other env variables, right?
```suggestion
    let master_key = std::env::var("MASTER_KEY").expect("Secrets Master_KEY env variable required");
```

Reply:
sure :+1: